handlers/officersHandler.go

```diff
@@ -0,0 +1,32 @@
+package handlers
+
+import (
+	"encoding/json"
+	"github.com/companieshouse/chs-delta-api/models"
+	"net/http"
+)
+
+// OfficerHandler offers a handler by which to publish a message onto a kafka topic.
+type OfficerHandler struct {
+}
+
+// NewOfficerHandler returns a OfficerHandler.
+func NewOfficerHandler() *OfficerHandler {
+	return &OfficerHandler{
+	}
+}
+
+// ServeHTTP accepts an incoming officer request via a POST method and validates and forwards to the XXX service.
```

Comment:
Should this be validates and publishes to an officer-delta kafka topic ?

Reply:
I suppose we didn't put that as it calls off to the service which will publish to the kafka topic

Reply:
Will it call to another service or just another method to publish to the topic ?

Reply:
We were thinking it would call off to a service layer that would then publish